src/devices/bus/hp_dio/hp98543.cpp

```diff
@@ -190,3 +192,5 @@ uint32_t dio16_98543_device::screen_update(screen_device &screen, bitmap_rgb32 &
 	}
 	return 0;
 }
+}
+}
```

Comment:
Since namespaces are big, I generally comment on the closing braces to make it clear what it should balance with, e.g.
```
} } // namespace bus::hp_dio
```